Package.swift

```diff
@@ -1,12 +1,12 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.5
 
 import PackageDescription
 
 
 let package = Package(
     name: "Apodini",
     platforms: [
-        .macOS(.v10_15)
+        .macOS(.v11)
```

Comment:
Not sure if the `.v12` version is available in the toolchain packaged with Xcode Beta 3 (at least its [documented](https://developer.apple.com/documentation/swift_packages/supportedplatform/macosversion/3857793-v12)),
but in any case you should be able to pass it as a string.

`.macs("12.0")`

This should allow you to remove all the `@available`s in the other places.

Reply:
Yes, beta 3 solves that issue. It also ships `AsyncStream` which was another missing piece. I'll try to finalize this ASAP. The last obstacle will probably be the outdated CI setup.